core/src/smartcontracts/isi/triggers/set.rs

```diff
@@ -1035,18 +1007,16 @@ impl<'block, 'set> SetTransaction<'block, 'set> {
 }
 
 /// WASM blob loaded with `wasmtime`
-#[derive(Clone)]
+#[derive(Clone, Copy, Serialize, Deserialize)]
 pub struct LoadedWasm {
-    /// Loaded Module
-    pub module: wasmtime::Module,
     /// Hash of original WASM blob on blockchain
     pub blob_hash: HashOf<WasmSmartContract>,
 }
 
 /// Same as [`Executable`](iroha_data_model::transaction::Executable), but instead of
 /// [`Wasm`](iroha_data_model::transaction::Executable::Wasm) contains WASM module as loaded
 /// by `wasmtime`
-#[derive(Clone)]
+#[derive(Clone, Serialize, Deserialize)]
 pub enum LoadedExecutable {
```

Comment:
Same as `LoadedWasm`, I don't feel the name reflects what this enum does anymore. For wasm contracts it just stores hash of the smart contract and for ISIs it stores the instructions themselves.

Maybe it makes sense to call it `DedupedExecutable` or `ExecutableRef`?

(btw, I think it should be mentioned somewhere that we store the wasm smart contracts in this roundabout way to do de-duplicate them)

Reply:
Agree that name is no longer very representative.
I like `ExecutableRef` since storing references doesn't imply deduplication of triggers.